pkg/image/bootc_disk.go

```diff
@@ -35,56 +31,41 @@ func NewBootcDiskImage(container container.SourceSpec) *BootcDiskImage {
 func (img *BootcDiskImage) InstantiateManifestFromContainers(m *manifest.Manifest,
 	containers []container.SourceSpec,
 	runner runner.Runner,
-	rng *rand.Rand) (*artifact.Artifact, error) {
```

Comment:
Interestingly, the `error` is always `nil`. I suggest keeping it as it is, because we want to use less panics in favor of errors, so I'm fairly sure we will have some errors to return in the future. :)